frontend/src/metabase/collections/components/CollectionSidebar/Bookmarks/Bookmarks.tsx

```diff
@@ -21,23 +23,47 @@ import {
 
 import { Bookmark, BookmarkableEntities, Bookmarks } from "metabase-types/api";
 
-interface LabelProps {
-  name: string;
-  display?: string;
-  type: BookmarkableEntities;
+interface BookmarkProps {
+  bookmark: Bookmark;
 }
 
 interface CollectionSidebarBookmarksProps {
   bookmarks: Bookmarks;
   deleteBookmark: (id: string, type: string) => void;
 }
 
-const Label = ({ display, name, type }: LabelProps) => {
-  const iconName = getIcon(display, type);
+interface IconProps {
+  name: string;
+  tooltip?: string;
+  isOfficial?: boolean;
+}
+
+const BookmarkIcon = ({ bookmark }: BookmarkProps) => {
+  const icon = BookmarkEntity.objectSelectors.getIcon(bookmark);
+  const isCollection = bookmark.type === "collection";
+  const isRegularCollection =
+    isCollection && PLUGIN_COLLECTIONS.isRegularCollection(bookmark);
+  const isOfficial = isCollection && !isRegularCollection;
+
+  const iconColor = isOfficial ? color("warning") : color("brand");
+  const iconOpacity = isOfficial ? 1 : 0.5;
+
+  return <BookmarkTypeIcon {...icon} color={iconColor} opacity={iconOpacity} />;
+};
+
+const Label = ({ bookmark }: BookmarkProps) => {
+  const icon = BookmarkEntity.objectSelectors.getIcon(bookmark);
+
   return (
     <LabelContainer>
-      <BookmarkTypeIcon name={iconName} />
-      {name}
+      {icon.tooltip ? (
+        <Tooltip tooltip={icon.tooltip}>
+          <BookmarkIcon bookmark={bookmark} />
+        </Tooltip>
+      ) : (
+        <BookmarkIcon bookmark={bookmark} />
+      )}
```

Comment:
[`Icon` also accepts a `tooltip` prop](https://github.com/metabase/metabase/blob/d373bf308c0eba6fc2b0bc84269a37b4e0608522/frontend/src/metabase/components/Icon.tsx#L159), you can just pass it down and it should work

Reply:
What a relief, thank you.